execute.c

```diff
@@ -19,8 +19,9 @@ void execute(char *c)
 		{
 				args[0] = c;
 				args[1] = NULL;
-				execve(c, args, NULL);
-				perror("execve");
+				/* execve(c, args, NULL); */
+				_execvp(c);
```

Comment:
why not just use execve and specify the path instead of using execvp(which isn't allowed)

Reply:
like this execve(c, args, NULL);

Reply:
I just found out the env part of execve, I'll make the appropriate changes